src/main/webapp/WEB-INF/tags/instructor/results/noResponsePanel.tag

```diff
@@ -27,7 +27,12 @@
                 <tbody>
                     <c:forEach items="${noResponsePanel.emails}" var="email">
                         <tr>
-                            <td>${fn:escapeXml(noResponsePanel.teams[email])}</td>
+                            <c:if test="${not empty noResponsePanel.instructorStatus[email]}">
+                                     <td><i>${fn:escapeXml(noResponsePanel.teams[email])}</i></td>
+                            </c:if>
+                            <c:if test="${empty noResponsePanel.instructorStatus[email]}">
```

Comment:
Completely agreed! My concern was that only checking for the element presence in the map, but not its value, might have looked like a bug to a future maintainer. As mentioned in the previous comment, can you also change the condition in the other `<c:if>`? Alternatively, let's switch to using `<c:choose>`, `<:when>` and `<c:otherwise>`, otherwise in future we might end up getting a bug where no team is shown at all. 

This PR should be fine after this! :)